helper_lib.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+ps_command=$(command -v ps)
+ps_args='-U root -u root -f'
```

Comment:
Can you add a comment here? Are we trying to filter by processes only owned by root?